Core/Object Arts/Dolphin/Installation Manager/DolphinCoreProduct.cls

```diff
@@ -100,7 +100,13 @@ contents
 	^contents!
 
 defaultSystemFolderColor
-	^RGB red: 41 green: 124 blue: 182!
+	"Reference RGB indirectly to eliminate hard dependency on Dolphin MVP Base.
```

Comment:
Another way to do this would be to just return an array of the R-G-B values here and use that to construct an RGB in the caller, but I don't think it is worth changing.